eng/common/scripts/Rest-Methods.ps1

```diff
@@ -0,0 +1,124 @@
+. "${PSScriptRoot}\logging.ps1"
```

Comment:
If we do end up keeping this script I'd suggest calling it "Invoke-RestAPI" or something like that.